opencensus/common/backports/__init__.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import six
+from six import raise_from
```

Comment:
@kshithijiyer would you clarify what's the desired state? Which "standard" are you trying to follow?

`import six` does follow the guidance http://google.github.io/styleguide/pyguide.html#224-decision, and now we are changing it to be against the guidance.

Reply:
I am just importing what is needed and not the entire thing. Didn't understand how is this against the guidance can you please explain?

Reply:
No I don't think so.

The guidance asks for importing the **module** instead of individual **class/function**. If you take a look at http://google.github.io/styleguide/pyguide.html#22-imports, you will see the statement "_Use import statements for packages and modules only, not for individual classes or functions_".

The guidance says do this:

```
from sound.effects import echo
...
echo.EchoFilter(input, output, delay=0.7, atten=4)
```

And don't do this:
```
from sound.effects.echo import EchoFilter
...
EchoFilter(input, output, delay=0.7, atten=4)
```

Reply:
Could you point out where else this mistake has happened?

Reply:
We cannot define mistakes without having clear rules.

My suggestion is to take a step back, clarify what is the desired state (e.g. having a set of rules, or simply saying we are going to follow http://google.github.io/styleguide/pyguide.html). Then we know what's "right" or "wrong" based on the rules.

Reply:
I prersonally feel that the best practice is to import whatever is needed instead of importing everything. Say you have 10 functions in a module and you just wish to use just 2 functions out of those 10 functions. In this case what's the point of importing the entire module instead you can just import 2 functions into the namespace?